### PR TITLE
refactor: merges dev-tickers

### PR DESCRIPTION
**Pull request by jurajpiar**
Merges dev tickers so that there is no inconsistency between local and production configuration.

Resolves [ROLLUP-182](https://rsklabs.atlassian.net/browse/ROLLUP-182)

### CODE REVIEW

File: core/bin/zksync_api/src/bin/dev_ticker_server.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    let _vlog_guard = vlog::init();
+
+    let opts = FeeTickerOpts::from_args();
+    if opts.sloppy {
+        vlog::info!("Fee ticker server will run in a sloppy mode.");
+    }
+
+    HttpServer::new(move || {
+        App::new()
+            // .app_data(web::Data::new(storage.clone()))
```
Comment (antomor) [inline core/bin/zksync_api/src/bin/dev_ticker_server.rs:32]: if we don't need the storage, I'd get rid of it

File: infrastructure/zk/src/docker.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ const IMAGES = [
     'keybase',
     'ci',
     'exit-tool',
-    'dev-liquidity-token-watcher',
+    'zk-environment',
```
Comment (antomor) [inline infrastructure/zk/src/docker.ts:14]: why do we add this here?

File: infrastructure/zk/src/config.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ const CONFIG_FILES = [
     'gateway_watcher.toml',
     'fee_ticker.toml',
     'misc.toml',
-    'dev_liquidity_token_watcher.toml',
+    'dev_ticker.toml',
```
Comment (antomor) [inline infrastructure/zk/src/config.ts:20]: How the dev_ticker differs from fee_ticker? Shouldn't we use one single file to configure the ticker?

**Review by antomor — CHANGES_REQUESTED** [review]
@jurajpiar I haven't test it yet and since we're not merging this to main, I plan to test it once we merge to the other branch. I've just left some questions, but overall looks good 👍🏼

File: infrastructure/zk/src/config.ts (hunk 3 above)
Comment (jurajpiar) [reply infrastructure/zk/src/config.ts:20]: fee_ticker contains configurations for production, whereas dev_ticker for ... well ... dev

File: infrastructure/zk/src/docker.ts (hunk 2 above)
Comment (jurajpiar) [reply infrastructure/zk/src/docker.ts:14]: oh I think that must have come from the rebase, I'll check. I certainly did not add it myself (that I know of ofc).

Comment (jurajpiar) [reply infrastructure/zk/src/docker.ts:14]: well, it does look like I added that accidentally, as there is no such line in either main or coingecko_2 branch. I'll remove it. I'm not sure how it got its way in tbh

Comment (jurajpiar) [reply infrastructure/zk/src/docker.ts:14]: done

File: core/bin/zksync_api/src/bin/dev_ticker_server.rs (hunk 1 above)
Comment (jurajpiar) [reply core/bin/zksync_api/src/bin/dev_ticker_server.rs:32]: done

**Pull request merged**